test/gtest/layernorm.hpp

```diff
@@ -24,7 +24,6 @@
  *
  *******************************************************************************/
 #include <miopen/miopen.h>
-#ifdef MIOPEN_BETA_API
```

Comment:
Please `#define MIOPEN_BETA_API 1` right above inclusion of miopen.h.

Reply:
fixed this

Reply:
[resolved]

Reply:
@seungmanhan Same as https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2481#discussion_r1390435324